components/sandbox/toolchain-host-operator/production/stone-prod-p01/toolchainconfig.yaml

```diff
@@ -72,4 +72,4 @@ spec:
       memberStatus:
         gitHubSecret:
           ref: member-operator-secret
-          accessTokenKey: github.access.token
```

Comment:
I think you deleted the last line of the file without noticing

Reply:
it's just the addition of a new line